.circleci/config.yml

```diff
@@ -742,12 +736,14 @@ jobs:
       - gitter_notify_failure_unless_pr
 
   chk_antlr_grammar:
-    <<: *base_buildpack_focal_small
+    <<: *base_cimg_small
     steps:
       - checkout
       - run:
           name: Install Java
-          command: apt -q update && apt install -y openjdk-16-jdk
+          command: |
+            sudo apt -q update
+            sudo apt install -y openjdk-16-jdk
```

Comment:
```
E: Unable to locate package openjdk-16-jdk
```
Not sure why this worked before, but apparently `openjdk-16` is [not a part of any Debian release](https://tracker.debian.org/pkg/openjdk-16); I get the same error when trying to install it locally. Can we bump this to `openjdk-17`?

Reply:
It did work and yeah, if version 17 works, it's good.

Reply:
Switched to version 17.